spec/acceptance/kubernetes_spec.rb

```diff
@@ -16,8 +16,8 @@
         case $facts['os']['family'] {
           'RedHat', 'CentOS':  {
             class {'kubernetes':
-              kubernetes_version => '1.22.0',
-              kubernetes_package_version => '1.22.0',
+              kubernetes_version => '1.28.2',
+              kubernetes_package_version => '1.28.2-1.1',
```

Comment:
Sure, this is just version for tests. The point was to update the version, that is available in the new repos. We should definitely update the default version, because the [current value](https://github.com/puppetlabs/puppetlabs-kubernetes/blob/main/manifests/init.pp#L604) is `1.10.2`.